package/doc/sphinx/source/documentation_pages/references.rst

```diff
@@ -204,6 +204,17 @@ please cite [Dima2004b]_.
    6564-6570. doi:`10.1021/jp037128y
    <https://doi.org/10.1021/jp037128y>`_
 
+If you calculate shape parameters using
+:meth:`~MDAnalysis.core.group.AtomGroup.quadrupole_moment`,
+:meth:`~MDAnalysis.core.group.ResidueGroup.quadrupole_moment`,
+:meth:`~MDAnalysis.core.group.SegmentGroup.quadrupole_moment`
+please cite [Gray1984]_.
+
+.. [Gray1984] Gray, C. G., Gubbins, K. E., & Joslin C. G. (1984). 
```

Comment:
See above comment on BibTex. @ojeda-e may be able to provide some guidance. :)

Reply:
What comment on BibTex? I didn't see a style guide specified so I did the best I could. I confirmed that the link rendered correctly locally.

Reply:
Yes, sorry, we have an open issue for BibTeX https://github.com/MDAnalysis/UserGuide/issues/212 — it's really new.

@ojeda-e please chime in!